orc8r/cloud/go/tools/combine_swagger/spec/combine.go

```diff
@@ -17,189 +17,49 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"regexp"
```

Comment:
There's no need for this combine_swagger/spec pkg anymore. Just put everything you need in the combine_swagger/main.go, and everything else should be in the lib code